src/graphql/resolvers.ts

```diff
@@ -357,6 +357,33 @@ const Query: QueryResolvers = {
 
     return results[0];
   },
+  cubeHealth: async () => {
+    const cube = await getObservationsCube();
+    const expectedDimensions = [
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/category",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/municipality",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/operator",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/period",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/aidfee",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/charge",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/energy",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/fixcosts",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/fixcostspercent",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/gridusage",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/total",
+      "https://energy.ld.admin.ch/elcom/electricityprice/dimension/product",
+      "https://cube.link/observedBy",
+    ];
```

Comment:
It this isn't supposed to change, maybe it would make sense to create this list only once outside of this function?